MANIFEST.in

```diff
@@ -1,9 +1,12 @@
 include *.py
 include *.txt
+include *.rst
+include .coveragerc
+include tox.ini
```

Comment:
Ah, the version of the ignore list [documented on PyPI](https://pypi.python.org/pypi/check-manifest) doesn't include '.travis.yml'.